docs/deprecation_policy.md

```diff
@@ -17,12 +17,13 @@ Details on the upstream support life cycles for Python and PostgreSQL are
 documented at [https://endoflife.date/python](https://endoflife.date/python) and
 [https://endoflife.date/postgresql](https://endoflife.date/postgresql).
 
-
 A Rust compiler is required to build Synapse from source. For any given release
 the minimum required version may be bumped up to a recent Rust version, and so
 people building from source should ensure they can fetch recent versions of Rust
 (e.g. by using [rustup](https://rustup.rs/)).
 
+The oldest supported version of SQLite is the version provided by
+[Debian oldstable](https://wiki.debian.org/DebianOldStable).
```

Comment:
Don't suppose there's a way to link to the latest such version?
https://packages.debian.org/oldstable/libsqlite3-0 seems to do it...?

Reply:
Thanks, that's a great idea.